src/components/Icon.jsx

```diff
@@ -1,5 +1,18 @@
+import Image from "next/image";
+import objectName from "../../public/navbar-logo.svg";
+
 const Icon = () => {
-  return <div>Icon</div>;
+  return (
+    <div className="flex">
+      <Image src={objectName} alt="Navbar-Logo" />
+      <div
+        className="font-extrabold text-white text-4xl -mb-4"
+        style={{ fontWeight: 2000, marginTop: "10px", marginLeft: "10px" }}
```

Comment:
avoid using standard css and stick to tailwind. u can just look up tailwind equivalents to these 3 styles and put them under className